projects/net-snmp/build.sh

```diff
@@ -27,6 +27,18 @@ $CXX $CXXFLAGS $WORK/snmp_pdu_parse_fuzzer.o \
       -Wl,-Bstatic -lcrypto -Wl,-Bdynamic -lm \
       -o $OUT/snmp_pdu_parse_fuzzer
 
+$CC $CFLAGS -c -Iinclude $SRC/snmp_parse_fuzzer.c -o $WORK/snmp_parse_fuzzer.o
```

Comment:
Same question here: does this change come from a Google employee or from you? In the latter case, please add a copyright line for your changes.

Reply:
The license header will have to stay as is - this is needed from OSS-Fuzz's perspective (all projects have the same license header).